ansible/roles/test/tasks/advanced-reboot.yml

```diff
@@ -145,6 +145,8 @@
     - include: advanced_reboot/reboot-image-handle.yml
       when: new_sonic_image is defined
 
+    - include: advanced_reboot/upgrade_mlnx_fw.yml
```

Comment:
It might be more logical to add this include to reboot-imagehhandle.yml but it works here too.

Reply:
still keeping this file. but I moved some condition check to the main file so that upgrade_mlnx_fw is not called all the time